### DISCUSSION TITLE
Trying to use the Sign In with Ethereum and I am seeing this error

### DISCUSSION THREAD

**Discussion by WiseMrMusa** [family/connectkit#198] [Q&A] [answered]
NotFoundError: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.

Call Stack
eval
node_modules/.pnpm/connectkit@1.2.2_ethers@5.7.2_react-dom@18.2.0_react-is@16.13.1_react@18.2.0_wagmi@0.12.10/node_modules/connectkit/build/index.es.js (9032:0)
React

**Comment by lochie** [comment]
Thanks for reporting. If you could please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) we can help debug further.

**Reply by WiseMrMusa** [reply, marked as answer]
I was using it with NextJS 13 experimental /app directory. I guess that was the problem because it was unstable. I had to start the code again with the stable /src and the SIWE feature is so good. Thanks for the feature

**Reply by hvdson** [reply]
any plans on having support for next 13 app directory? I'm running into this error message as well

**Reply by lochie** [reply]
Here's an example app that uses App Router that may help resolve any problems:
https://github.com/m1guelpf/nextjs13-connectkit-siwe

**Comment by LI-YONG-QI** [comment]
> I was using it with NextJS 13 experimental /app directory. I guess that was the problem because it was unstable. I had to start the code again with the stable /src and the SIWE feature is so good. Thanks for the feature

Any ways are available for using Next.js 13?
I meet the problem as well

**Reply by lochie** [reply]
Here's an example app that uses App Router that may help resolve any problems:
https://github.com/m1guelpf/nextjs13-connectkit-siwe
